examples/src/main/java/io/numaproj/numaflow/examples/source/simple/SimpleSource.java

```diff
@@ -80,4 +81,9 @@ public long getPending() {
     public List<Integer> getPartitions() {
         return Sourcer.defaultPartitions();
     }
+
+    @VisibleForTesting
```

Comment:
This seems like an anti-pattern. VisibleForTesting. The class itself should be easy to use, not requiring some backdoor just for testing.